tcms/telemetry/templates/telemetry/include/filters.html

```diff
@@ -43,7 +43,7 @@
             {% trans "Test Plan" %}
         </label>
         <div class="col-md-3">
-            <select name="test_plan" id="id_test_plan" class="form-control selectpicker">
+            <select name="test_plan" id="id_test_plan" class="form-control selectpicker" multiple>
```

Comment:
finally when you have multiple selection the search button sends an API request with the appropriate ORM field/values which doesn't know anything about multiple selection.